zerver/lib/digest.py

```diff
@@ -378,6 +393,16 @@ def bulk_get_digest_context(
         context["new_channels"] = new_streams
         context["new_streams_count"] = new_streams_count
 
+        context[
+            "message_content_disabled_by_realm"
+        ] = not realm.message_content_allowed_in_email_notifications
```

Comment:
Should we not check the user setting as well, like we do in missed message email notifications? See `message_content_allowed_in_missedmessage_emails`.